pkg/apb/rhcc_registry.go

```diff
@@ -57,39 +57,37 @@ func (r RHCCRegistry) LoadSpecs() ([]*Spec, int, error) {
 	r.log.Debug("RHCCRegistry::LoadSpecs")
 	var specs []*Spec
 
-	imageList, err := r.LoadImages("*-apb")
+	imageList, err := r.LoadImages("\"*-apb\"")
 	if err != nil {
 		return []*Spec{}, 0, err
 	}
 
 	numResults := imageList.NumResults
 	r.log.Debug("Found %d images in RHCC", numResults)
 	for _, image := range imageList.Results {
-		spec, err := r.imageToSpec(image)
-		if err != nil {
-			return []*Spec{}, 0, err
+		if spec := r.imageToSpec(image); spec != nil {
+			specs = append(specs, spec)
 		}
-		specs = append(specs, spec)
 	}
 
 	return specs, numResults, nil
 }
 
-func (r RHCCRegistry) imageToSpec(image *Image) (*Spec, error) {
+func (r RHCCRegistry) imageToSpec(image *Image) (*Spec) {
```

Comment:
I think travis doesn't like you have parenthesis around *Spec in the return statement.